proposals/lip-0054.md

```diff
@@ -100,24 +100,24 @@ Since these technical requirements are not straightforward, the recovery command
 | Name                                          | Type   | Value        |
 |-----------------------------------------------|--------|--------------|
 | **Module**                                    |        |              |
-| `MODULE_ID_INTEROPERABILITY`                  | uint32 | 64           |
-| `MODULE_ID_TOKEN`                             | uint32 | TBD          |
+| `MODULE_ID_INTEROPERABILITY`                  | bytes  | TBD          |
+| `MODULE_ID_TOKEN`                             | bytes  | TBD          |
 | **Store Prefix**                              |        |              |
 | `STORE_PREFIX_CHAIN_DATA`                     | bytes  | 0x8000       |
 | `STORE_PREFIX_TERMINATED_STATE`               | bytes  | 0xc000       |
 | `STORE_PREFIX_TERMINATED_OUTBOX`              | bytes  | 0xb000       |
 | **Status**                                    |        |              |
-| `CHAIN_TERMINATED`                            | uint32 | 2            |
+| `CHAIN_STATUS_TERMINATED`                            | uint32 | 2            |
 | `CCM_STATUS_OK`                               | uint32 | 0            |
 | `CCM_STATUS_RECOVERED`                        | uint32 | 4            |
 | **Command ID**                                |        |              |
-| `COMMAND_ID_STATE_RECOVERY`                   | uint32 | 4            |
-| `COMMAND_ID_MESSAGE_RECOVERY`                 | uint32 | 5            |
-| `COMMAND_ID_STATE_RECOVERY_INITIALIZATION`    | uint32 | 6            |
-| `COMMAND_ID_MESSAGE_RECOVERY_INITIALIZATION`  | uint32 | 7            |
-| `CROSS_CHAIN_COMMAND_ID_TRANSFER`             | uint32 | 0            |
+| `COMMAND_ID_STATE_RECOVERY`                   | bytes  | 0x0004       |
+| `COMMAND_ID_MESSAGE_RECOVERY`                 | bytes  | 0x0005       |
+| `COMMAND_ID_STATE_RECOVERY_INITIALIZATION`    | bytes  | 0x0006       |
+| `COMMAND_ID_MESSAGE_RECOVERY_INITIALIZATION`  | bytes  | 0x0007       |
+| `CROSS_CHAIN_COMMAND_ID_TRANSFER`             | bytes  | 0x0000       |
 | **Other**                                     |        |              |
-| `MAINCHAIN_ID`                                | uint32 | 1            |
+| `MAINCHAIN_ID`                                | bytes | 0x00000001    |
```

Comment:
All length constants are not defined in this LIP either:

`HASH_LENGTH`, `ADDRESS_LENGTH`, CHAIN_ID_LENGTH`, `MODULE_ID_LENGTH` (`MODULE_NAME_MAX_LENGTH with the new terminology..)

Reply:
I removed all constant s from the LIP